lib/junk_drawer/rails/bulk_updatable.rb

```diff
@@ -8,15 +8,23 @@ module JunkDrawer
   # module to allow bulk updates for `ActiveRecord` models
   module BulkUpdatable
     ATTRIBUTE_TYPE_TO_POSTGRES_CAST = {
+      boolean: '::boolean',
+      date: '::date',
       datetime: '::timestamp',
+      float: '::float',
       hstore: '::hstore',
-      boolean: '::boolean',
+      integer: '::int',
+      json: '::json',
       jsonb: '::jsonb',
+      decimal: '::decimal',
```

Comment:
Why no `string: '::text'`?

Reply:
Because it worked without it? I think Postgres assumes everything is a string by default, so it doesn't need to be specified. For the rest of these, when trying to assign the value as `null`, it would complain that you gave it a string when it expected a `::decimal` or a `::time` or whatnot, so in these cases we need to tell it to treat `null` as a decimal value with `null::decimal`. Kind of weird, but it did the trick.

Reply:
oh interesting. mmk ![bitmoji](https://render.bitstrips.com/v2/cpanel/c7485269-9eb6-4c88-ab79-859e3eff4c96-e487a764-d181-48ad-a0fc-98fd8ad2e737-v1.png?transparent=1&palette=1&width=246)